cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/monitoring/viewSsoSessions.jsp

```diff
@@ -20,13 +20,80 @@
 --%>
 <%@include file="/WEB-INF/view/jsp/default/ui/includes/top.jsp"%>
 
-<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/highlight.min.js"></script>
```

Comment:
You removed the highlight library, but I don't see the removal of the specific CSS styles.